cloud/gc_dns.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python
```

Comment:
Why use the hardcoded /usr/bin/python over the more flexible alternative what like Eric used? The path /usr/bin/env will always exist except in really rare cases.